src/types/page.urls.ts

```diff
@@ -7,6 +7,7 @@ export enum urlParams {
 }
 
 export enum URL_QUERY_PARAM {
+  COMPANY_NUM = "companyNumber",
```

Comment:
I'd name this COMAPNY_NUMBER

Reply:
I did try that but Sonar raised it as a code smell due to there being another COMPANY_NUMBER already in the file so had to come up with another name in keeping with the other Enum IS_PSC